pkg/dsl/definition/package.go

```diff
@@ -70,7 +70,7 @@ func NewPackageDiscover(config *rest.Config) (*PackageDiscover, error) {
 		pkgKinds: make(map[string][]VersionKind),
 	}
 	if err = pd.RefreshKubePackagesFromCluster(); err != nil {
-		return nil, err
+		return pd, err
```

Comment:
will the pd be nil?

Reply:
`pd` will be nil, i will add some check and prevent it